README.md

```diff
@@ -2,40 +2,45 @@
 
 This repository contains a reference implementation of the Commerce Extensibility Framework. It includes sample code for extending the following services:
 
-- Pricing
-- Shipping Calculator
-- Tax Calculator
-- Tax Service
+- [Domain Extensions for Cart Calculate API](#domain-extensions-for-cart-calculate-api):
```

Comment:
Reorganized readme a little bit so it reflects the same structure as we have in the docs